identity/app_secret.go

```diff
@@ -1,10 +1,6 @@
 package identity
 
-import (
-	identity2 "github.com/TankerHQ/identity-go/v3"
-)
-
 // DEPRECATED: use github.com/TankerHQ/identity-go/v3
 
-const AppSecretSize = identity2.AppSecretSize
-const AppPublicKeySize = identity2.AppPublicKeySize
+const AppSecretSize = 64
+const AppPublicKeySize = 32
```

Comment:
should these be exposed?

Reply:
I think they were before, and this folder has been left here only so as to keep the new version compatible with previous ones... So I'd say until we switch major, we need to leave this here :/